### DISCUSSION TITLE
Color mode only for night mode

### DISCUSSION THREAD

**Discussion by LennardPlay** [claytonjn/hass-circadian_lighting#242] [Ideas]
Hi :) 
Is it possible to use a lamp in ct mode during the day, but set it to a dark red at night? I heard dark red doesn't disrupt your sleep pattern that much.

**Comment by claytonjn** [comment]
That isn't currently possible, although I personally have this same use-case and achieve it by disabling CL and using a different automation to set the color/brightness.

**Reply by LennardPlay** [reply]
Thanks for your answer. That actually sounds great and I'm a bit ashamed I haven't simply though of that.
